src/pages/AddJob.jsx

```diff
@@ -42,6 +43,73 @@ export default function AddJob() {
     }
   );
 
+  useEffect(() => {
+    let hasdraft = window.localStorage.getItem("draft");
```

Comment:
actually lots of websites may use localStorage.
I would suggest using more specific name like `REFUGEE_ONE_JOB_DRAFT`, to make sure this do not conflict with other websites